README.md

```diff
@@ -1 +1,73 @@
-# 2mobile4
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
```

Comment:
- [ ] You have to edit the README file template with your information and information about your project. Right now you have just the original template. Kindly address this issue to make your README file more professional.